catalog/nsili/catalog-nsili-transformer/src/main/java/org/codice/alliance/nsili/transformer/DAGConverter.java

```diff
@@ -1373,6 +951,16 @@ public static String getValueString(Collection<Serializable> collection) {
         return thumbnail;
     }
 
+    private void addDescription(Metacard metacard, String description) {
+        Attribute descAttr = metacard.getAttribute(CoreAttributes.DESCRIPTION);
```

Comment:
Kind of a nit, but you can also just use the Core interface directly vs using CoreAttributes